tools/config.py

```diff
@@ -3,15 +3,15 @@
 
 # Update with the Meilisearch version TAG you want to build the AMI with
 
-MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.27.1'
+MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.28.0'
 
 # Update with the AMI id that you want to publish after TESTING
 
-PUBLISH_IMAGE_ID = 'ami-0d5f3e203a56d644e'
+PUBLISH_IMAGE_ID = 'ami-0c5702d1a96f2d307'
 
 # Update with the AMI name that you want to unpublish/delete worldwide
 
-DELETE_IMAGE_NAME = 'Meilisearch-v0.27.0-Debian-10'
+DELETE_IMAGE_NAME = 'Meilisearch-v0.25.2-Debian-10'
```

Comment:
Yes and no, actually, when I release a new version, I delete the older one.
But last time, I chose to delete v0.27 because we released a patch right after (v0.27.1 then v0.27.2) and I think v0.25.2 will be more useful than v0.27 for users.